LIPs/LIP-25.md

```diff
@@ -180,9 +180,9 @@ It's established already that intial `owner` this governance system will be the
 
 As aforementioned delay will initially be enforced through social convention but expectations for social rules should still be set.
 
-The proposal is that the Livepeer Inc multisig uses a delay of 1.5x the current unbonding period when staging contract upgrades or parameter updates. This value should be used as the `_delay` argument when calling `stage()`. 
+The proposal is that the Livepeer Inc multisig uses a 9 round delay (unbonding period plus two rounds) when staging contract upgrades or parameter updates. This value should be used as the `_delay` argument when calling `stage()`. 
```

Comment:
Suggestion:

"9 round delay (unbonding plus two rounds)" -> "delay equal to the unbonding period plus two rounds"

Motivation for the suggestion: Make sure that it is clear that the delay is always defined based on the unbonding period (as long as this LIP is in effect) and to avoid an interpretation where the delay is initially set to 9 rounds based on the unbonding period without any expectation of it always being defined based on the unbonding period.

Reply:
Fixed and flattened into a single commit 77da151